App.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "./index.css";
+import ContactUserCard from "./Components/ContactUserCard";
+import Contacts from "./Components/Contacts";
+import NavBar from "./Components/NavBar";
+import Post from "./Components/Post";
+import Posts from "./Components/Posts";
+import UserProfile from "./Components/UserProfile";
+
+const App = () => {
+  return (
+    <div className="App">
+    <NavBar></NavBar>
```

Comment:
Hey Alex, it looks like none of your tests are passing. Here's a couple notes/tips on how to fix up your code. 

Components (written in JSx tags), for now, do not have opening and closing tags, like HTML. They're written like <NavBar />. They also are not nested within each other all in the <App /> component. For example, all of your <Post /> components should be imported and used within your <Posts /> component. <Posts /> is then imported into <App />. I think, with all the work we've done over the last week, that you can refactor your code and resubmit correctly.